lib/SetAPI/SetAPIImpl.py

```diff
@@ -187,9 +186,10 @@ def get_differential_expression_matrix_set_v1(self, ctx, params):
         # ctx is the context object
         # return variables are: result
         # BEGIN get_differential_expression_matrix_set_v1
-        ws = Workspace(self.workspaceURL, token=ctx["token"])
-        demsi = DifferentialExpressionMatrixSetInterfaceV1(ws)
-        result = demsi.get_differential_expression_matrix_set(ctx, params)
+        ws = Workspace(self.workspace_url, token=ctx["token"])
+        set_interface = SetInterfaceV1(ws)
+        result = set_interface.get_set(params)
```

Comment:
all `get` queries now go straight to the SetInterfaceV1 class  with their parameters